### PR TITLE
STL 1935 Fixes to the content page

### PR DESCRIPTION
**Pull request by hamo225**
# Short Description
- Some styling fixes to the InfoTaxReturnForPensionersPage

# Changes
- Separates out styles for the content page 
- Makes changes to some text translations and moves a link
- Fixes some element widths and font sizes in the content page

# Feedback
- Any?

# How to review this Pull Request
[Link to Confluence](https://digitalservice4germany.atlassian.net/wiki/spaces/STL/pages/117637157/Development+Process#Approve-or-comment) (internal)

### CODE REVIEW

File: webapp/client/src/components/SecondaryAnchorButton.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ import styled from "styled-components";
 import addPlausibleGoal from "../lib/helpers";
 
 const AnchorSecondary = styled.a`
-  padding: 1rem 1.25rem calc(1rem - 4px) 1.25rem; /* The calculation subtracts the border-bottom height. We need a border-bottom for the focus state. */
+  padding: 18px 24px 18px 24px;
```
Comment (JulianRoesner) [inline webapp/client/src/components/SecondaryAnchorButton.js:6]: I guess this was copied from earlier? I think that the padding leads to a problem when it is in focus state (it toggles).

**Review by JulianRoesner — APPROVED** [review]
LGTM. However, I think the padding of the SecondaryButton has a problem in the focus state; you might need to have a second look

File: webapp/client/src/lib/translations.js [hunk 2]
```diff
@@ -441,7 +440,7 @@ const translations = {
       listItemFourHeading: "Schritt 4: Steuererklärung ausfüllen",
 
       listItemFour:
-        "Nach Erhalt des Freischaltcodes können Sie mit der <activationLink>Steuererklärung</activationLink> beginnen. Dafür melden Sie sich auf der Webseite unter „Ihre Steuererklärung“ mit den entsprechenden Daten an. Aber aufgepasst: bevor Sie mit der vereinfachten Steuererklärung loslegen, sollten Sie alle erforderlichen Informationen, wie zum Beispiel Belege, vorliegen haben. Und damit Sie nichts vergessen und perfekt vorbereitet sind, gibt es noch eine nützliche Checkliste zum Herunterladen.",
+        "Nach Erhalt des Freischaltcodes können Sie mit der Steuererklärung beginnen. Dafür melden Sie sich auf der Webseite unter <activationLink>„Ihre Steuererklärung“</activationLink> mit den entsprechenden Daten an.",
```
[the remark this answers is not in the document]
Comment (hamo225) [reply webapp/client/src/lib/translations.js:443]: Yes. Change in the text to link from and removing text.

File: webapp/client/src/components/SecondaryAnchorButton.js (hunk 1 above)
Comment (hamo225) [reply webapp/client/src/components/SecondaryAnchorButton.js:6]: Your right...

**Pull request merged**